app/angular/tslint.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
We could move this up, if there's more typescript in the codebase? Would that work?

Reply:
Well, this one's got some Angular specific stuff, such as `"node_modules/codelyzer"`